infra/ansible/inventory.yml

```diff
@@ -151,6 +153,7 @@ all:
           npm_lts_version: fermium
           frontend_scheme: https
           frontend_csp_connect_src: https://noembed.com
+          frontend_csp_frame_ancestors: https://www.youtube.com
```

Comment:
Does adding `"chrome-extension:"` for allowed ancestors allow it for any chrome extension? Wouldn't this be a problem?

Reply:
Yes it would allow it for any extension. But I don't think excluding extensions provides strong security guarantees anyway, and an extension may still rewrite these headers to work around this policy. 
We could also define the exact extension ids in the policy, however that would make it cumbersome for developers as a local extension uses a random generated id.
So I would find this change an acceptable trade-off, as long as the extension relies on this intermediate iframe.

Reply:
> Additional question: do you know whether this intermediate iframe ("tournesol-iframe.html") is still required?
If I remember correctly, this was used as a workaround to synchronize the distinct storages used by the extension and the domain "tournesol.app". However the new "Rate now" button seems to work without this trick. So I am a bit lost

Unfortunately I don't remember exactly the challenges we faced when we created the modal, and I don't know if it's still required. We could investigate that.

Is it possible that the intermediate iframe was here to ease to simply split the different pieces of code (html / css / javascript)?